dashboard/app/helpers/plc/tasks_helper.rb

```diff
@@ -0,0 +1,6 @@
+module Plc::TasksHelper
+  def options_for_plc_tasks
+    #Probably better to hardcode this - this list isn't going to change that much
+    [['Plc::LearningResourceTask', 'Learning Resource'],['Plc::ScriptCompletionTask', 'Script Completion Task']]
```

Comment:
I think 'Learning Resource' was meant to be the label:

<img width="279" alt="screen shot 2016-03-01 at 4 25 00 pm" src="https://cloud.githubusercontent.com/assets/413693/13442438/310500ba-dfca-11e5-94ed-4fb09f333430.png">